pkg/apis/autoscaling/register.go

```diff
@@ -233,9 +243,9 @@ var (
 		InitialScaleAnnotationKey,
 		GroupName + "/initialScale",
 	}
-
-	MaxScaleAnnotation = kmap.KeyPriority{
-		MaxScaleAnnotationKey,
+	ScaleMaxAnnotation = kmap.KeyPriority{
+		ScaleMaxAnnotationKey,
+		DeprecatedMaxScaleAnnotationKey,
 		GroupName + "/maxScale",
```

Comment:
I wouldn't add `Deprecated...` variables, just include the additional key as we've done with `/maxScale`